tools/mc-java/src/test/java/io/spine/tools/mc/java/gradle/AnnotationProcessorConfigurationTest.java

```diff
@@ -26,55 +26,46 @@
 
 package io.spine.tools.mc.java.gradle;
 
-import com.google.common.testing.NullPointerTester;
 import io.spine.testing.UtilityClassTest;
 import io.spine.tools.mc.java.gradle.given.StubProject;
 import org.gradle.api.Project;
 import org.gradle.api.artifacts.Configuration;
 import org.gradle.api.artifacts.ConfigurationContainer;
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 import static com.google.common.truth.Truth.assertThat;
 import static io.spine.tools.gradle.ConfigurationName.annotationProcessor;
-import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNull;
 
-@DisplayName("`PreprocessorConfig` should")
+@DisplayName("`AnnotationProcessorConfiguration` should")
 class AnnotationProcessorConfigurationTest
         extends UtilityClassTest<AnnotationProcessorConfiguration> {
 
-    private Project project;
-    private ConfigurationContainer projectConfigs;
-    private Configuration preprocessorConfig;
+    private static Project project = null;
+    private static ConfigurationContainer configurations = null;
+    private static Configuration preprocessorConfig = null;
 
     AnnotationProcessorConfigurationTest() {
         super(AnnotationProcessorConfiguration.class);
     }
 
-    @BeforeEach
-    void setUp() {
-        project = StubProject.createFor(getClass()).get();
-        projectConfigs = project.getConfigurations();
-        preprocessorConfig = projectConfigs.getByName(annotationProcessor.value());
+    @BeforeAll
+    static void createProject() {
+        project = StubProject.createFor(AnnotationProcessorConfigurationTest.class).get();
+        configurations = project.getConfigurations();
+        preprocessorConfig = configurations.getByName(annotationProcessor.value());
     }
 
     @Test
-    @DisplayName("return annotation processor config if it exists")
-    void returnAnnotationProcessorConfigIfItExists() {
-        Configuration returnedConfig = AnnotationProcessorConfiguration.findOrCreateIn(project);
-        assertEquals(preprocessorConfig, returnedConfig);
-    }
-
-    @Test
-    @DisplayName("create and return annotation processor config if it does not exist")
+    @DisplayName("create annotation processor config if it does not exist")
     void createAndReturnAnnotationProcessorConfigIfItDoesNotExist() {
-        projectConfigs.remove(preprocessorConfig);
-        assertNull(projectConfigs.findByName(annotationProcessor.value()));
+        configurations.remove(preprocessorConfig);
+        assertNull(configurations.findByName(annotationProcessor.value()));
```

Comment:
Let's replace `assertNull` with `Truth.assertThat().isNull`. We're usually migrating the code to Truth assertions while touching up some tests.